### PR TITLE
Fix check process is running

### PR DESCRIPTION
**Pull request by googlielmo**
- exclude `avahi` from `ps -ef` output

### CODE REVIEW

File: common-tests/src/main/resources/commontests/common.tests.bom [hunk 1]
```diff
@@ -193,7 +193,7 @@ brooklyn.catalog:
         brooklyn.config:
           command:
             $brooklyn:formatString:
-              - "ps -ef | grep '%s' | grep -v 'dhclient'"
+              - "ps -ef | grep '%s' | grep -v 'dhclient' | grep -v 'avahi'"
```
Comment (m4rkmckenna) [inline common-tests/src/main/resources/commontests/common.tests.bom]: How about 
```bash
ps -ef | grep '%s' | grep -vE 'avahi|dhclient'
```

**Review by m4rkmckenna — COMMENTED** [review]
Minor comment feel free to ignore

**Comment by googlielmo** [comment]
@m4rkmckenna thanks for the comment, it's much more readable using extended regexes

**Review by m4rkmckenna — APPROVED** [review]
LGTM

**Pull request merged**